packages/cli/src/templates/package.json

```
@@ -6,14 +6,19 @@
   "type": "module",
   "scripts": {
     "build": "mimic compile",
```

I think this should work
```suggestion
    "build": "codegen && compile",
```